app/src/main/res/layout/passengerlist_items.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.cardview.widget.CardView
+    xmlns:android = "http://schemas.android.com/apk/res/android"
+    xmlns:tools = "http://schemas.android.com/tools"
+    android:layout_width = "match_parent"
+    android:layout_height = "wrap_content"
+    xmlns:app = "http://schemas.android.com/apk/res-auto"
+    android:layout_marginTop = "35dp"
+    android:layout_marginStart = "25dp"
+    android:layout_marginEnd = "25dp"
+    app:cardCornerRadius = "15dp"
+    app:cardElevation="30dp"
+    >
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width = "match_parent"
+        android:layout_height = "wrap_content"
+        android:padding = "15dp"
+        >
+
+
+            <androidx.appcompat.widget.AppCompatImageView
+                android:id = "@+id/imageView"
+                android:layout_width = "wrap_content"
+                android:layout_height = "wrap_content"
+                android:src = "@drawable/profile"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintEnd_toStartOf="@id/passenger"
+                app:layout_constraintTop_toTopOf="parent"
+                app:layout_constraintBottom_toBottomOf="parent"
+                />
+
+            <androidx.appcompat.widget.LinearLayoutCompat
```

Comment:
here try to add constraint with imageview to the right using chains ,take refernce from [here](https://stackoverflow.com/questions/47155926/constraintlayout-margins)